### DISCUSSION TITLE
Cannot see Add to Dataset button in the UI

### DISCUSSION THREAD

**Discussion by SinghCoder** [langfuse/langfuse#866] [Support] [answered]
This is in the Langfuse cloud server.

The documentation [here](https://langfuse.com/docs/datasets) explicitly mentions:

```
Create items from production data

In the UI, use + Add to dataset on any production trace.
```

![image](https://github.com/langfuse/langfuse/assets/32142363/88bfc5d6-937c-4902-8c04-19f09b4c30e8)

**Comment by marcklingen** [comment, marked as answer]
Currently, observations (spans, events, generations) can be added to datasets. If you navigate to a single observation on trace (1) and sufficient rights in a project, you should see the button (2).

![CleanShot 2024-01-11 at 15 27 59@2x](https://github.com/langfuse/langfuse/assets/2834609/6f0b907a-dc73-45af-a575-7b61e645449b)

Thanks for the hint, I've updated the docs to make this more clear.

**Comment by SinghCoder** [comment]
Ahh, thanks! Yup, this works!

**Reply by marcklingen** [reply]
Np, happy to help
